tests/network/wireguard.pm

```diff
@@ -20,6 +20,19 @@ use utils;
 use lockapi;
 use mmapi 'wait_for_children';
 
+sub start_wgquick {
+    foreach my $dev (@_) {
+        # Start wg-quick with some debug output on failure
+        if (script_run("systemctl start wg-quick\@$dev") != 0) {
+            script_run("systemctl status wg-quick\@$dev");
+            save_screenshot();
```

Comment:
We're using serial console (`select_serial_terminal()`), not sure if `save_screenshot()` will work.

Reply:
Good point and as of now you see the serial output anyway in openQA. I removed the `save_screenshot` routines.